<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeySetwidget</class>
 <widget class="QWidget" name="KeySetwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1135</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1135</width>
    <height>620</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_caption">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_report_rate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>45</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Report Rate</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_125">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>75</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>125Hz/8ms</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_250">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>75</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>250Hz/4ms</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_500">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>500Hz/2ms</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1000">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1000Hz/1ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sleep_mode">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>160</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set sleep mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_usb_mode">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>USB connection mode</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_usb_mode">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>220</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>set a time(s)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_backlight">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>285</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>set a time(s)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_wireless">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>350</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>set a time(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_backlight">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>255</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Backlighting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wireless">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>320</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.4G/Bluetooth</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_usb_mode">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_backlight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>285</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_bluetooth">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_keyboard">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>400</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Restore Keyboard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
